modules/hcp-create-infraenv.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-deploy/hcp-deploy-ibm-power.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-create-infraenv_{context}"]
+= Creating infrastructure environment resources
+
+For heterogeneous node pools, you must create an `infraenv` custom resource (CR) for each architecture. For example, for node pools with `x86_64` and `ppc64le` architectures, create an `infraenv` CR for `x86_64` and `ppc64le`.
+
+.Procedure
+
+Before proceeding, make sure that the operating system (OS) images for both `x86_64` and `ppc64le` architectures are added to the `AgentServiceConfig` resource. After this, you can use the `infraenv` resources to get the minimal ISO image.
```

Comment:
We generally don't put content between ".Procedure" and the first step. So line 13 could be moved above `.Procedure." If it's really important you could make it a note or an IMPORTANT admonition. 

https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs